assets/messages.yml

```diff
@@ -216,7 +216,22 @@ messages:
         However, this issue is {color:#FF5722}*Invalid*{color}.
 
         This is an account issue. We do not have the tools to help you with this issue on this bug tracker.
-        Please have a look at the help article "[Why have I been banned from Minecraft?|https://help.minecraft.net/hc/en-us/articles/360052618531-Why-have-I-been-banned-from-Minecraft-]" and the [Community Standards|https://www.minecraft.net/en-us/community-standards].
+        Please have a look at the help article "[Why have I been banned from Minecraft?|https://help.minecraft.net/hc/en-us/articles/4408964729869-Why-Have-I-Been-Banned-from-Minecraft-]" and the [Community Standards|https://www.minecraft.net/en-us/community-standards].
+        To request a case review of a ban, please contact Mojang Support through [this page|https://help.minecraft.net/hc/en-us/requests/new?ticket_form_id=360003469452].
+        
+        %quick_links%
+      fillname: []
+    - project:
+        - mc
+      name: Banned
+      shortcut: ban
+      message: |-
+        *Thank you for your report!*
+        However, this issue is {color:#FF5722}*Invalid*{color}.
+        
+        This is an account issue. We do not have the tools to help you with this issue on this bug tracker.
+        Please have a look at the help article "[Why have I been banned from Minecraft?|https://help.minecraft.net/hc/en-us/articles/4408964729869-Why-Have-I-Been-Banned-from-Minecraft-]" and the [Community Standards|https://www.minecraft.net/en-us/community-standards].
```

Comment:
"Why Have I Been Banned from Minecraft?" seems to be mostly about Bedrock edition at the moment, the screenshot is from Bedrock edition and the text says:
> when they attempt to sign into Minecraft on any platform (non-Java Edition)

So maybe we should omit that link for now?

Reply:
Ah, I missed that. I'll change it. Thanks :)